web/static/js/app.js

```diff
@@ -11,11 +11,22 @@
 //
 // If you no longer want to use a dependency, remember
 // to also remove its path from "config.paths.watched".
-import "phoenix_html"
+import 'phoenix_html';
+import accessibleAutocomplete from 'accessible-autocomplete';
 
 // Import local files
 //
 // Local files can be imported directly using relative
 // paths "./socket" or full ones "web/static/js/socket".
 
 // import socket from "./socket"
+//
+// const repos = ['learn-elixir', 'time', 'tudo'];
+
+accessibleAutocomplete({
```

Comment:
could we make use of this?

https://github.com/alphagov/accessible-autocomplete#progressive-enhancement

Reply:
For sure, yes. The current implementation includes a select menu so we won't even have to write new code. :+1:

Reply:
This change has been made now, please re-review when you get a chance :pray: